profile/README.md

```diff
@@ -14,32 +14,29 @@
 Miden is the edge blockchain that grants applications the power to scale with public and private transactions. It aims at builders who want to create an ecosystem for private asset management and OTC-trading.
```

Comment:
I'm not 100% sure about mentioning the use case here - or at least mentioning just the use case.  We may want to mention a bigger vision here as well.